external/googleapis/CMakeLists.txt

```diff
@@ -157,18 +157,22 @@ foreach (proto ${EXTERNAL_GOOGLEAPIS_PROTO_FILES})
 endforeach ()
 
 include(ExternalProject)
+
 ExternalProject_Add(
     googleapis_download
     EXCLUDE_FROM_ALL ON
     PREFIX "${CMAKE_BINARY_DIR}/external/googleapis"
     URL ${GOOGLE_CLOUD_CPP_GOOGLEAPIS_URL}
     URL_HASH SHA256=${GOOGLE_CLOUD_CPP_GOOGLEAPIS_SHA256}
-             # Scaffolding for patching googleapis after download. For example:
-             # PATCH_COMMAND patch -p1 --input=/v/external/googleapis.patch
-             # NOTE: This should only be used while developing with a new
-             # protobuf message. No changes to `PATCH_COMMAND` should ever be
-             # committed to the main branch.
-    PATCH_COMMAND ""
+    PATCH_COMMAND
```

Comment:
FYI, the formatting was probably confused because `PATCH_COMMAND` is not a known argument, you can add them here:

https://github.com/googleapis/google-cloud-cpp/blob/67de7b3da3c4f52f25fa24d693854c7945792b22/.cmake-format.py#L20-L47

Reply:
It looks like `PATCH_COMMAND` is there.